README.DropDownFilter.md

```diff
@@ -0,0 +1,40 @@
+# Drop down filter
+Enable users to filter a list view at run time, with various options that can be selected from a drop-down
+
+## Features
+* Filter items by an attribute
+* Filter items by XPath
+* Select a filter from a list of options
+* Set a default filter option
+
+## Dependencies
```

Comment:
no need to specify dependency per widget, should only be in the general readme

Reply:
Done